devito/ops/utils.py

```diff
@@ -24,3 +24,9 @@
 namespace['ops_dat_d_p'] = lambda i: '%s_d_p' % i
 namespace['ops_dat_d_m'] = lambda i: '%s_d_m' % i
 namespace['ops_dat_name'] = lambda i: '%s_dat' % i
+namespace['ops_par_loop_range_name'] = 'par_loop_range'
+namespace['ops_par_loop'] = 'ops_par_loop'
+namespace['ops_arg_dat'] = 'ops_arg_dat'
```

Comment:
This (and the next one) can be safely made into functions (like other ops api calls)